src/utils/utils.ts

```diff
@@ -167,3 +167,5 @@ export const utf8ToBytes32 = (str: string): string => {
       .padStart(64, '0')
   )
 }
+
+export const cleanRequest = (query: any) => JSON.stringify(query).replace(/\s+/g, ' ')
```

Comment:
Can you just add a little note in a comment above describing how it cleans the request?